libs/payments/ui/src/lib/client/components/StripeWrapper.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+import { loadStripe, StripeElementsOptions } from '@stripe/stripe-js';
+import { Elements } from '@stripe/react-stripe-js';
+
+interface StripeWrapperProps {
+  amount: number;
+  currency: string;
+}
+
+export function StripeWrapper({ amount, currency }: StripeWrapperProps) {
+  // TODO - Load from config
+  const stripePromise = loadStripe(
+    'pk_test_VNpCidC0a2TJJB3wqXq7drhN00sF8r9mhs'
+  );
+
+  const options: StripeElementsOptions = {
```

Comment:
Do we need to include locale in `options`?

Reply:
For now the default value, `auto`, uses the browser language, which matches our current app logic.